datascience/util.py

```diff
@@ -22,3 +24,30 @@ def percentile(p, arr=None):
     if arr is None:
         return lambda arr: percentile(p, arr)
     return np.percentile(arr, p, interpolation='higher')
+
+def plotCDFarea(rbound, lbound=None, mean=0, sd=1):
+    """Plots a normal curve with specified parameters and area below curve shaded
+    between ``lbound`` and ``rbound``.
+
+    Args:
+        ``rbound`` (numeric): right boundary of shaded region
+
+        ``lbound`` (numeric): left boundary of shaded region; by default is negative infinity
+
+        ``mean`` (numeric): mean/expectation of normal distribution
+
+        ``sd`` (numeric): standard deviation of normal distribution
+    """
+    llabel = lbound
+    if lbound is None:
+        lbound = sd * -3.5 + mean
+        llabel = "-$\infty$"
+    z = np.arange(sd * -3.5 + mean, sd * 3.5 + mean, 0.1)
+    plt.plot(z, stats.norm.pdf(z, loc=mean, scale=sd), color='k', lw=1)
+    w = np.arange(lbound, rbound + .1, 0.1)
+    plt.fill_between(w, stats.norm.pdf(w, loc=mean, scale=sd), color='gold')
```

Comment:
Are there more descriptive name than `w` and `z`? It seems nitpicky right now but someone down the road will be really happy if variable names are good.